core/src/com/group66/game/screens/SplitGameScreen.java

```diff
@@ -0,0 +1,343 @@
+package com.group66.game.screens;
```

Comment:
You and Jeroen mentioned this already, but indeed in the future a common (abstract) GameScreen should be used to build upon instead of duplicating code. (just stating it for completeness)